### PR TITLE
Update welcome page

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**

### CODE REVIEW

File: dist/war/getstarted/index.html [hunk 1]
```diff
@@ -1,90 +1,156 @@
...
+  <meta name="description" content="">
+  <meta name="author" content="Activeeon">
+  <link rel="shortcut icon" href="assets/image/favicon.ico">
 
-    <title>Get started with ProActive</title>
+  <title>Get started with ProActive</title>
 
-    <link href="assets/css/bootstrap.css" rel="stylesheet">
-    <link href="assets/css/custom.css" rel="stylesheet">
+  <link rel="stylesheet" href="assets/css/bootstrap4.1.3//bootstrap.min.css">
```
Comment (fviale) [inline dist/war/getstarted/index.html:13]: you did not add this file in your pull request.

You need to to git add -f assets/css/bootstrap4.1.3//bootstrap.min.css I think

File: dist/war/getstarted/assets/css/custom.css [hunk 2]
```diff
@@ -8,3 +10,203 @@ body {
...
+
+.form-style-1 li {
+    padding: 0;
+    display: block;
+    list-style: none;
+    margin: 10px 0 0 0;
+}
+
+.form-style-1 label {
+    margin: 0 0 0 0;
```
Comment (medou-boushab) [inline dist/war/getstarted/assets/css/custom.css:29]: margin: 0;
will apply to all four sides

File: dist/war/getstarted/assets/css/custom.css [hunk 3]
```diff
@@ -8,3 +10,203 @@ body {
...
+.form-style-1 input[type=date],
+.form-style-1 input[type=datetime],
+.form-style-1 input[type=number],
+.form-style-1 input[type=search],
+.form-style-1 input[type=time],
+.form-style-1 input[type=url],
+.form-style-1 input[type=email],
+.form-style-1 input[type=password],
+.form-style-1 input[type=hidden]
+
```
Comment (medou-boushab) [inline dist/war/getstarted/assets/css/custom.css:47]: I guess you need a colon here?

Comment (nebilbenmabrouk) [reply dist/war/getstarted/assets/css/custom.css:47]: Actually, many resources were missing. I added them all.

File: dist/war/getstarted/index.html (hunk 1 above)
Comment (nebilbenmabrouk) [reply dist/war/getstarted/index.html:13]: Required resources are added correctly now

Comment (amouhoub) [reply dist/war/getstarted/index.html:13]: @nebil, watch out for `//bootstrap.min` !, remove the extra slash

**Pull request merged**